zeek_log_transport.sh

```diff
@@ -17,6 +17,48 @@ export PATH="/sbin:/usr/sbin:$PATH"		#Note that cron does _NOT_ include /sbin in
 
 default_user_on_aihunter='dataimport'
 
+get_send_logs() {
+        ##These names were found at https://docs.zeek.org/en/master/script-reference/log-files.html
+        declare -a zeek_logs=("conn" "dce_rpc" "dhcp" "dnp3" "dns" "ftp" "http" "irc" "kerberos" "modbus" "modbus_register_change" "mysql" "ntlm" "ntp" "radius" "rdp" "rfb" "sip" "smb_cmd" "smb_files" "smb_mapping" "smtp" "snmp" "socks" "ssh" "ssl" "syslog" "tunnel" "files" "ocsp" "pe" "x509" "netcontrol" "netcontrol_drop" "netcontrol_shunt" "netcontrol_catch_release" "openflow" "intel" "notice" "notice_alarm" "signatures" "traceroute" "known_certs" "known_hosts" "known_modbus" "known_services" "software" "barnyard2" "dpd" "unified2" "unknown_protocols" "weird" "weird_stats" "broker" "capture_loss" "cluster" "config" "loaded_scripts" "packet_filter" "print" "prof" "reporter" "stats" "stderr" "stdout"  "conn-summary")
+        send_logs=()
+
+	destination=$1
+	dest_log_trans=$2
+	local_dest_dir=$3
+
+        `rsync $destination:$dest_log_trans $local_dest_dir`
+
+	##Parse file for logs skip headerlines (indicated by :) and commented lines (indicated by #)
+        request_logs=`cat $local_dest_dir | grep -v : | grep -v #`
+
+	##TODO check if the request logs is set to all if so we should set the send_logs to the zeek_logs variable
+	if [[ " ${zeek_logs[*]} " == *"all"* ]]; then
```

Comment:
I'm not sure if the spaces around `${zeek_logs[*]}` are intended. I don't think the `*"all"*` needs to be quoted. In general, things in `[[ ]]` don't need to be quoted. But strings with spaces or variables in `[ ]` or elsewhere do need to be quoted.